packages/builder/src/pages/builder/portal/users/users/index.svelte

```diff
@@ -237,6 +251,25 @@
     <Body>Add users and control who gets access to your published apps</Body>
   </Layout>
   <Divider />
+  {#if showUserLimitAlert}
+    <InlineAlert
+      type="error"
+      onConfirm={() => {
+        window.open("https://budibase.com/pricing/", "_blank")
```

Comment:
need to change this to check for account portal access and link to the upgrade page if so